src/main/java/org/osiam/resources/controller/ServiceProviderConfigController.java

```diff
@@ -23,20 +23,20 @@
 
 package org.osiam.resources.controller;
 
-import java.util.HashSet;
-import java.util.Set;
-
-import org.osiam.resources.helper.RequestParamHelper;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseBody;
 
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.util.HashSet;
+import java.util.Set;
 
 @Controller
```

Comment:
Do want to make this a `@RestController`, too, and remove the `@ResponseBody` below?